en/development/testing.rst

```diff
@@ -300,6 +300,14 @@ additional ``<testsuite>`` sections to your application's **phpunit.xml** file::
 Any additional test suites added to the ``<testsuites>`` element will
 automatically be run when you use ``phpunit``.
 
+NOTE : If you are using <testsuites> to run tests on plugins that you installed with composer, then the plugin's composer.json file should add the fixture namespace to the autoload setting.  Example...
+
```

Comment:
Preformatted blocks are preceded with `::` Right now your code block will not render.